package.json

```diff
@@ -303,6 +303,7 @@
     "commander": "2.20.3",
     "css-loader": "3.6.0",
     "csstype": "2.6.21",
+    "dotenv": "^16.3.1",
```

Comment:
can we version lock this rather than `^`

Reply:
this is a dev dependency with no transient dependencies that's actively maintained. Nothing within the code looks suspicious and socket checks look fine to me. I'm good with the addition just blocked on locking the version is all.

Reply:
Done! I guess `npm install --save-dev` automatically adds the `^`.